src/components/PostComment.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from "react";
+import { useParams  } from "react-router-dom";
+import { useNavigate  } from "react-router-dom";
+import "../styling/postcomment.css";
+import { PostCommentByArticleId } from "../app";
+
+
+function PostComment() {
+  const { article_id } = useParams();
+  const [newUsername, setUsername] = useState("");
+  const [inputBody, setBody] = useState("");
+
+  const navigate = useNavigate();
+
+  const newComment = {
+    username: newUsername,
+    body: inputBody,
+  };
+
+const handleSubmit = (event) => {
+    event.preventDefault();
+
+    PostCommentByArticleId(article_id, newComment)
+    navigate(`/articles/${article_id}`)
```

Comment:
What happens if the api request fails. We should have some isError functionality so that that user gets a warning if something goes wrong.